### ISSUE TITLE
Why add panGestureRecognizer to spreadsheetView

### ISSUE THREAD

**Issue by keshiim** [bannzai/SpreadsheetView#174]
```swift
 [tableView, columnHeaderView, rowHeaderView, cornerView, overlayView].forEach {
            addGestureRecognizer($0.panGestureRecognizer)
            #if swift(>=3.2)
            if #available(iOS 11.0, *) {
                $0.contentInsetAdjustmentBehavior = .never
            }
            #endif
        }
```

`addGestureRecognizer($0.panGestureRecognizer)` Could you tell me why. sometime I received [Spreadsheetview _panGestureRecognizer]  selector can not found from xcode

**Comment by keshiim** [comment]
I`m ready to convert your swift Project to OC version.  But I have met the difficulties mentioned above. Please help me with review it. Thanks
link: https://github.com/keshiim/ZMJGanttChart

**Comment by keshiim** [comment]
oh! I fixed it by adding below code
```objc
- (BOOL)isKindOfClass:(Class)aClass {
    if (@available(iOS 11.0, *)) {
        return [super isKindOfClass:aClass];
    } else {
        return [self.rootView isKindOfClass:aClass];
    }
}
```
